src/junitFormatter.js

```diff
@@ -0,0 +1,60 @@
+const { writeFileSync } = require('fs');
+var xml2js = require('xml2js');
+var os = require('os');
+
+// Private: Returns a failure XML element, with required properties and inner
+// text.
+function formattedTestError(test) {
+  return {
+    $: {
+      message: test.message,
```

Comment:
I think this should just be `test.errorMessage`; I'm a lil' worried about our cute cross emoji being inside an XML attribute.

I've checked what an rspec failure looks like and it's like this:

```
<failure message="Unable to find visible css &quot;trix-editor&quot;" type="Capybara::ElementNotFound">Failure/Error: find(&apos;trix-editor&apos;).click.set copy

Capybara::ElementNotFound:
  Unable to find visible css &quot;trix-editor&quot;
/Users/jalada/.rvm/gems/ruby-2.6.6/gems/capybara-3.32.0/lib/capybara/node/finders.rb:302:in `block in synced_resolve&apos;
/Users/jalada/.rvm/gems/ruby-2.6.6/gems/capybara-3.32.0/lib/capybara/node/base.rb:83:in `synchronize&apos;
/Users/jalada/.rvm/gems/ruby-2.6.6/gems/capybara-3.32.0/lib/capybara/node/finders.rb:291:in `synced_resolve&apos;
/Users/jalada/.rvm/gems/ruby-2.6.6/gems/capybara-3.32.0/lib/capybara/node/finders.rb:52:in `find&apos;
/Users/jalada/.rvm/gems/ruby-2.6.6/gems/capybara-3.32.0/lib/capybara/session.rb:759:in `block (2 levels) in &lt;class:Session&gt;&apos;
/Users/jalada/.rvm/gems/ruby-2.6.6/gems/capybara-3.32.0/lib/capybara/dsl.rb:58:in `block (2 levels) in &lt;module:DSL&gt;&apos;
./spec/features/xxxxxxxx.rb:49:in `block (4 levels) in &lt;top (required)&gt;&apos;</failure>
```

Reply:
As discussed, swap it over; long one in the body.